v1/recommend.yaml

```diff
@@ -0,0 +1,76 @@
+info:
+  version: '1.0.0'
+  title: Recommendation API
+  description: Recommendation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache license, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /recommendation/translation/{from_lang}{/seed_article}:
+    get:
+      tags:
+        - Recommendation
+      summary: Recommend articles for translation.
+      description: |
+        Recommends articles to be translated from the source
+        to the domain language.
+
+        See more at [Recommendation API documentation](https://meta.wikimedia.org/wiki/Recommendation_API)
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
+      produces:
+        - applicaiton/json
+      parameters:
+        - name: from_lang
+          in: path
+          description: The source language code
+          type: string
+          required: true
+        - name: seed_article
+          in: path
+          description: The article to use as a search seed
+          type: string
+          required: false
+        - name: count
+          in: query
+          description: The max number of articles to return
+          type: integer
+          required: false
+          default: 24
+      responses:
+        '200':
+          description: the list of articles recommended for translation
+          schema:
+            $ref: '#/definitions/translation_result'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: '{{options.host}}/{domain}/v1/translation/articles/{from_lang}/{seed_article}'
+      x-monitor: false
+definitions:
+  translation_result:
+    type: object
+    properties:
+      count:
+        type: integer
+        description: the number of recommendations returned
+      articles:
```

Comment:
Normally for lists of articles we use `items`? Maybe better to stick with a pattern?